Firmware/Marlin_main.cpp

```diff
@@ -6511,9 +6511,9 @@ SERIAL_PROTOCOLPGM("\n\n");
     */
     case 73: //M73 show percent done, time remaining and time to change/pause
     {
-        if(code_seen('P')) print_percent_done_normal = code_value();
+        if(code_seen('P')) print_percent_done_normal = code_value_uint8();
         if(code_seen('R')) print_time_remaining_normal = code_value();
```

Comment:
please rename `code_value` to `code_value_float` so that we don't run into such an issue in the future.

Reply:
We already have code_value_float with a workaround for E notation. How should we proceed?